debian/rules

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/make -f
+%:
+	dh $@ --with python3 --buildsystem=pybuild
+
+override_dh_auto_clean:
+	dh_auto_clean
+	rm -f debian/key-mapper.service
+
+override_dh_auto_install:
+	dh_auto_install
+	# remove the systemd-service in /usr/lib/systemd and replace it with dh-systemd
+	rm -rf debian/key-mapper/usr/lib/systemd/
+	cp data/key-mapper.service debian/key-mapper.service
+	# move udev-rule from /etc to /lib for packaging
+	mkdir -p debian/key-mapper/lib
+	mv debian/key-mapper/etc/udev debian/key-mapper/lib/udev
```

Comment:
should this be changed in setup.py?

Reply:
I don't know whats best practice for pip-packages. Debian packages use `/lib/udev` and lintian complains if it detects a udev-rule in a different directory.

Reply:
yeah lets change that in setup.py instead https://unix.stackexchange.com/a/333714

Reply:
Sounds good. Do you make the commit? I'll rebase this PR afterwards.